pkg/cloud/services/networking/port.go

```diff
@@ -161,6 +161,10 @@ func (s *Service) GetOrCreatePort(eventObject runtime.Object, clusterName string
 		record.Warnf(eventObject, "FailedCreatePort", "Failed to create port %s: %v", portName, err)
 		return nil, err
 	}
+	if err = s.replaceAllAttributesTags(eventObject, portResource, port.ID, tags); err != nil {
```

Comment:
If `len(tags) == 0`, then would this OpenStack call be necessary? We may be able to be save a call per port creation in the cases where tags are not specified on the instance.